db/schema.rb

```diff
@@ -10,375 +12,397 @@
-    t.bigint "byte_size", null: false
-    t.string "checksum", null: false
-    t.datetime "created_at", null: false
-    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
+  create_table 'active_storage_blobs', force: :cascade do |t|
+    t.string 'key', null: false
+    t.string 'filename', null: false
+    t.string 'content_type'
+    t.text 'metadata'
+    t.bigint 'byte_size', null: false
+    t.string 'checksum', null: false
+    t.datetime 'created_at', null: false
+    t.index %w[key], name: 'index_active_storage_blobs_on_key', unique: true
   end
 
-  create_table "attendances", force: :cascade do |t|
-    t.bigint "meeting_id"
-    t.bigint "user_id"
-    t.string "special_duties"
-    t.boolean "completed_hours"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["meeting_id"], name: "index_attendances_on_meeting_id"
-    t.index ["user_id"], name: "index_attendances_on_user_id"
+  create_table 'attendances', force: :cascade do |t|
+    t.bigint 'meeting_id'
+    t.bigint 'user_id'
+    t.string 'special_duties'
+    t.boolean 'completed_hours'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index %w[meeting_id], name: 'index_attendances_on_meeting_id'
+    t.index %w[user_id], name: 'index_attendances_on_user_id'
   end
 
-  create_table "box_items", force: :cascade do |t|
-    t.integer "box_id", null: false
-    t.integer "researched_by_id"
-    t.boolean "added_to_box"
-    t.integer "created_by_id", null: false
-    t.integer "updated_by_id", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.bigint "inventory_type_id", null: false
-    t.datetime "researched_at"
-    t.integer "quantity", default: 1
-    t.index ["box_id"], name: "index_box_items_on_box_id"
-    t.index ["created_by_id"], name: "index_box_items_on_created_by_id"
-    t.index ["inventory_type_id"], name: "index_box_items_on_inventory_type_id"
-    t.index ["researched_by_id"], name: "index_box_items_on_researched_by_id"
-    t.index ["updated_by_id"], name: "index_box_items_on_updated_by_id"
+  create_table 'box_items', force: :cascade do |t|
+    t.integer 'box_id', null: false
+    t.integer 'researched_by_id'
+    t.boolean 'added_to_box'
+    t.integer 'created_by_id', null: false
+    t.integer 'updated_by_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'inventory_type_id', null: false
+    t.datetime 'researched_at'
+    t.integer 'quantity', default: 1
+    t.index %w[box_id], name: 'index_box_items_on_box_id'
+    t.index %w[created_by_id], name: 'index_box_items_on_created_by_id'
+    t.index %w[inventory_type_id], name: 'index_box_items_on_inventory_type_id'
+    t.index %w[researched_by_id], name: 'index_box_items_on_researched_by_id'
+    t.index %w[updated_by_id], name: 'index_box_items_on_updated_by_id'
```

Comment:
This is a generated file, right? I think this will churn unnecessarily, and we should tell Rubocop to ignore it